data/gatherings.yml

```diff
@@ -182,14 +182,13 @@ gatherings:
       - label: "RSVP Required"
         price: "Free"
     lead_text: >-
-      This OpenShift Commons Gathering will be held in Buenos Aires at the <a href="https://www.wscc.com/" target="_blank">Washington State Convention Center</a>!
+      Donde se congregan usuarios, socios, clientes, colaboradores y líderes de proyectos de desarrollo para colaborar y trabajar codo a codo con OpenShift.!
```

Comment:
Did you want this line ending in `.!`?

Reply:
yes, push it live, and i will correct it in the next round

Reply:
👍